src/common/components/with-store-subscription.tsx

```diff
@@ -14,7 +14,7 @@ export type WithStoreSubscriptionDeps<T> = {
 };
 
 export function withStoreSubscription<P extends WithStoreSubscriptionProps<S>, S>(
-    WrappedComponent: React.ComponentType<P>,
+    WrappedComponent: React.ComponentType<React.PropsWithChildren<P>>,
```

Comment:
this changes are as per react-types for react18 changes, this values are updated with codemod script provided in react18 documentation.
This script automatically converts all old types to new types i.e. react18 types